pkg/migrations/sql/preinstall/003-tables.sql

```diff
@@ -94,13 +94,16 @@ CREATE TABLE SCHEMA_CATALOG.metric (
     id SERIAL PRIMARY KEY,
     metric_name text NOT NULL,
     table_name name NOT NULL,
+    table_schema name NOT NULL DEFAULT 'SCHEMA_DATA',
+    series_table name DEFAULT NULL, -- NULL signifies the series table is named the same as metric table name
```

Comment:
I couldn't relate, what is a `series_table` here?

Reply:
Table that holds series data. Will add a comment to make it more clear.